.github/workflows/build-release.yml

```diff
@@ -200,8 +200,16 @@ jobs:
       run: |
         cd ${{ matrix.template.folder }}
         echo "Current directory: $(pwd)"
-        echo "Directory contents:"
-        cargo build --release --locked
+        case "${{ matrix.template.folder }}" in 
+          "frontier-evm"|"blue-l2")
```

Comment:
openzeppelin-evm and pop-evm too? i think they all based off of frontier

Reply:
Yes, they are based off frontier but both OZ and POP added the needed code for enabling metadata-hash, whereas frontier does not have it.
- POP-EVM: https://github.com/r0gue-io/evm-parachain/blob/a95b83b256dfd1e24c631edcbb5983b2d808b5cb/runtime/build.rs#L1
- OZ-EVM: https://github.com/OpenZeppelin/polkadot-runtime-templates/blob/2bfcbf94d5d24a7b2177904da74ca84c8607689b/evm-template/runtime/build.rs#L6